### ISSUE TITLE
Ar gps conditions question

### ISSUE THREAD

**Issue by gbo999** [dmbfm/unity-ar-gps-location-issues#61]
Hi Daniel,

I have built a simple app to see how it works. I build the app twice.at the first time i took coordinates from google maps for 50 meter from my home-didn't see nothing. The second time i put the coordinates at my house and i saw the prefab i made.

 So i want to ask  does the environment need to be lighten? How far from the prefab i am suppose to be in order see the it ?how big the objects need to be?(because it was really small. 

A picture where i put the prefab in my house coordinates:

![image](https://user-images.githubusercontent.com/31617142/84127574-21396c00-aa48-11ea-850f-7282ee2e65a8.png)

**Comment by horatiu665** [comment]
50 meters away might just be super tiny to see, like pixels wide. Try creating a straight line of like 25 objects, starting from the house, and placed 2 meters from each other, and see how small they get as the distance increases.

**Comment by dmbfm** [comment]
> **So i want to ask does the environment need to be lighten?
Yes, AR in general will only work well if the environment is well lighted. 

> How far from the prefab i am suppose to be in order see the it ?how big the objects need to be?(because it was really small. and does it suppose to take time to load when i am near the location?**
> what else i should do in order to see it 50 meter from my house-because i didn't see the prefab from there.

The visibility distance will be determined only by the camera's far clip distance. You can change it's value in the inspector panel
for the ARCamera in the scene.

The size of a 3D object will, of course, be smaller the farther the object is from the device.
